h/admin/services/user.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h import models
+from h.api.search.index import BatchIndexer
+from h.util.user import userid_from_username
+
+
+class UserRenameError(Exception):
+    pass
+
+
+class RenameUserService(object):
```

Comment:
Ah, OK. I think I was originally imagining a service with a slightly broader scope that handled all user admin actions. So the way I was originally intending to approach this was 1) Create a `UserAdminService` and move the existing logic for handling user actions here in one PR, 2) Add the rename user functionality in a separate PR.

If you approach this as you have done here, adding a service to rename users first, would you later create a separate service for the different user admin tasks or rename this service and broaden its scope to include other tasks and move the logic for `admin_users_delete` into it?